app/src/main/res/values/strings.xml

```diff
@@ -39,7 +39,7 @@
     <string name="skip_short_break">Skip Short Break</string>
     <string name="start_long_break">Start Long Break</string>
     <string name="skip_long_break">Skip Long Break</string>
-    <string name="skip_break">Skip Break</string>
```

Comment:
This String has it's usages, I wonder why you removed it.

Reply:
Ohh sorry. It got removed when I rebased it.